docs/apis/subsystems/files/index.md

```diff
@@ -194,7 +194,7 @@ function mod_myplugin_pluginfile(
     }
 
     // For a plugin which does not specify the itemid, you may want to use:
-    $itemid = null to make your code more consistent.
+    $itemid = null; //to make your code more consistent.
```

Comment:
I'd suggest leaving this as a comment as it is an _alternative_ to the code above.
```suggestion
    // $itemid = null;
```